packages/elements/src/components/ino-dialog/readme.md

```diff
@@ -58,32 +55,48 @@ class MyComponent extends Component {
+margin at top for the background scrim and escape for dialog close. It's not recommenended to use this option with `--ino-dialog-width` and `--ino-dialog-height`.
+
+### Closing actions
+You can mark elements as "action elements" by providing a `data-ino-dialog-action="action-name"` attribute.
+On user clicks, the dialog checks whether the target is a dialog action and, if true, emits a `close` event with `event.detail = "action-name"`.
+
+Subsequently, listen to the `close` Event and check the `event.detail` to retrieve the users action.
 
 <!-- Auto Generated Below -->
 
 
 ## Properties
 
-| Property | Attribute | Description                     | Type      | Default |
-| -------- | --------- | ------------------------------- | --------- | ------- |
-| `open`   | `open`    | Opens the dialog if set to true | `boolean` | `false` |
+| Property      | Attribute     | Description                                                                                                                                                            | Type      | Default     |
+| ------------- | ------------- | ---------------------------------------------------------------------------------------------------------------------------------------------------------------------- | --------- | ----------- |
+| `attachTo`    | `attach-to`   | The target element the dialog should be attached to. If not given, the dialog is a child of the documents body. Note: This property is immutable after initialization. | `string`  | `undefined` |
+| `dismissible` | `dismissible` | Close the dialog on pressing the ESC key or clicking outside of the dialog.                                                                                            | `boolean` | `undefined` |
+| `fullwidth`   | `fullwidth`   | Defines a full width dialog sliding up from the bottom of the page.                                                                                                    | `boolean` | `undefined` |
+| `open`        | `open`        | Opens the dialog if set to true                                                                                                                                        | `boolean` | `false`     |
 
 
 ## Events
 
-| Event        | Description                                       | Type               |
-| ------------ | ------------------------------------------------- | ------------------ |
-| `openChange` | Emits an event upon opening or closing the dialog | `CustomEvent<any>` |
+| Event   | Description                            | Type                  |
+| ------- | -------------------------------------- | --------------------- |
+| `close` | Emits an event upon closing the dialog | `CustomEvent<string>` |
 
 
 ## Slots
 
 | Slot        | Description           |
 | ----------- | --------------------- |
-| `"content"` | content of the dialog |
-| `"footer"`  | footer of the dialog  |
-| `"header"`  | header of the dialog  |
```

Comment:
Why do you prefer a single slot instead of three separate slots?
I know that this is a lot more flexible but wouldn't that speak against our design system concept?

Reply:
Yeah, as long as we don't have any styling, I would remove the unused slots. They add complexity:
* why is the footer height 20% when it doesn't have any styling?
* why do we need css vars to set the height on a slotted content?

In my opinion, we should use the `ino-dialog` as a generic base component and follow the concept of composition, meaning either:
* for instance `ino-confirmation-dialog` as a child in the dialog providing a special functionality for confirmations,  or a `ino-dialog-wizard` for fullwidth dialogs.
* or style classes for containers in a global.scss being imported.